core/src/com/etheller/warsmash/viewer5/handlers/w3x/simulation/data/CAbilityData.java

```diff
@@ -155,6 +158,7 @@ private void registerCodes() {
 		this.codeToAbilityTypeDefinition.put(War3ID.fromString("AIma"), new CAbilityTypeDefinitionItemManaRegain());
 		this.codeToAbilityTypeDefinition.put(War3ID.fromString("AIat"), new CAbilityTypeDefinitionItemAttackBonus());
 		this.codeToAbilityTypeDefinition.put(War3ID.fromString("AIab"), new CAbilityTypeDefinitionItemStatBonus());
+		this.codeToAbilityTypeDefinition.put(War3ID.fromString("AIrc"), new CAbilityTypeDefinitionItemStatBonus());
```

Comment:
This one says `ItemStatBonus` on the end, was it meant to say `ItemReincarnation` ?